tests/testsuite/run.rs

```diff
@@ -1416,6 +1416,23 @@ fn default_run_workspace() {
     p.cargo("run").with_stdout("run-a").run();
 }
 
+#[cargo_test]
+fn print_env_verbose() {
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+                [package]
+                name = "a"
+                version = "0.0.1"
+            "#,
+        )
+        .file("src/main.rs", r#"fn main() {println!("run-a");}"#)
+        .build();
+
+    p.cargo("run -vv").with_stderr("LD_LIBRARY_PATH").run();
```

Comment:
`LD_LIBRARY_PATH` won't be set on Windows. I'd recommend a different one like `CARGO_PKG_NAME` for the test case. 

You can use `[..]` as a wildcard for matching.

```suggestion
    p.cargo("run -vv")
        .with_stderr_contains("[RUNNING] [..]CARGO_PKG_NAME=a[..]")
        .run();
```

Reply:
@weihanglo beat me to the review with the same comment 😊. I'll close this one.